Kernel/ACPI/Initialize.cpp

```diff
@@ -21,10 +22,15 @@ UNMAP_AFTER_INIT void initialize()
     if (!rsdp.has_value())
         return;
 
-    if (feature_level == AcpiFeatureLevel::Enabled)
-        Parser::initialize<DynamicParser>(rsdp.value());
-    else
-        Parser::initialize<Parser>(rsdp.value());
+    auto facp = StaticParsing::find_table(rsdp.value(), "FACP");
+    if (!facp.has_value())
+        return;
+    auto facp_table = Memory::map_typed<Structures::FADT>(facp.value());
+    u8 irq_line = facp_table->sci_int;
+
+    Parser::initialize(rsdp.value(), facp.value(), irq_line);
+    VERIFY(Parser::the());
+    Parser::the()->set_feature_level(kernel_command_line().acpi_feature_level());
```

Comment:
As far as I'm aware the acpi feature level is constant for the lifetime of the Kernel, so there's no reason to have a setter for it, can we just pass it in the constructor instead/